Mathlib/NumberTheory/Ostrowski.lean

```diff
@@ -28,6 +28,52 @@ absolute values on a number field up to equivalence. (TODO)
 ring_norm, ostrowski
 -/
 
+namespace Filter
+
+-- ## Preliminary lemmas on limits
```

Comment:
```suggestion
/-! ## Preliminary lemmas on limits -/
```
Please use this syntax so that this comments will appear in the docs.

Same below.

Reply:
Done!